integrationTests/__snapshots__/custom-parser.test.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Doesn't override parser when not set 1`] = `
+exports[`Doesn't override parser when not set [ESLint<6] 1`] = `
```

Comment:
seems like this is failing on CI?

Reply:
The snapshot was locally updated by jest 26. It seems they strip it for a reason. I'm checking the release notes.

Reply:
Nevermind I did something wrong. They all pass!